packages/integration-tests/src/client/storage.ts

```diff
@@ -1,7 +1,7 @@
 import { Storage, StorageKey } from '@logto/node';
 import { Nullable } from '@silverhand/essentials';
 
-export class DummyStorage implements Storage {
+export class MemoryStorage implements Storage {
```

Comment:
What is kept in storage could be called dummy data. Technically we are creating storage here regardless of the content being kept, either useful or not.

Reply:
Not refer to the data kept in the storage.
The storage itself is a dummy object because we set `persistAccessToken` to `false`, and this `Storage` will not be used.